src/components/grapher/GrapherDots.vue

```diff
@@ -1,40 +1,37 @@
 <template>
   <g class="grapher-dots--dots-container">
-    <circle
-      v-for="dot in stuntSheetDots"
+    <Dot
+      v-for="(dot, index) in stuntSheetDots"
       :key="`${dot.x}-${dot.y}-dots--dot`"
       class="grapher-dots--dot"
+      data-test="grapher-dots--dot"
+      :transform="`translate(${dot.xAtBeat(beat - 1)}, ${dot.yAtBeat(
+        beat - 1
+      )})`"
       :cx="dot.xAtBeat(beat - 1)"
       :cy="dot.yAtBeat(beat - 1)"
-      r="0.7"
-      data-test="grapher-dots--dot"
+      :dotTypeIndex="dot.dotTypeIndex"
+      :label="dotLabels[index]"
+      :labeled="showDotLabels"
```

Comment:
This doesn’t need to be a property, Dot.vue is able to grab this value since it also has access to the global vuex store!

Reply:
I was thinking about keeping this a prop so that if we show the dot appearance in a different place we can not show its label regardless of the store for example in the dot type pane 
<img width="263" alt="image" src="https://user-images.githubusercontent.com/29964426/102672125-dca03300-4144-11eb-8774-d37a105690ee.png">

Reply:
@MichaelTamaki are you referring to the `:labeled` property?
@adhamrait are you referring to the `:label` property?

Reply:
Yeah, I'm referring to `labelled`! 

But I think I understand where you're coming from... You mean keep this flexible so that we can show a `Dot` right next to the Dot Type label in the right side bar, as well as use `Dot` on the `Grapher`. Makes sense!